mm2src/mm2_main/src/rpc.rs

```diff
@@ -205,7 +205,7 @@ async fn process_single_request(ctx: MmArc, req: Json, client: SocketAddr) -> Re
 
 #[cfg(not(target_arch = "wasm32"))]
 async fn rpc_service(req: Request<Body>, ctx_h: u32, client: SocketAddr) -> Response<Body> {
-    const NON_ALLOWED_CHARS: &[char] = &['<', '>', '&'];
+    const NON_ALLOWED_CHARS: &[char] = &['Ò'];
```

Comment:
we won't be able to return a clean url from rpc response like this `wc:a4b99985b6144e6fb20ae089483a9f9f1d2da9dd99a40a3241d7261690bc95ed@2?symKey=2376ed73d73459732621982bceb6ec70d267bcbc4bed1dc8d6b5bd278bf9e9fb&relay-protocol=irn&expiryTimestamp=1730810382`, since the rule will try to replace those chars with the escaped version

Reply:
> we won't be able to return a clean url from rpc response like this `wc:a4b99985b6144e6fb20ae089483a9f9f1d2da9dd99a40a3241d7261690bc95ed@2?symKey=2376ed73d73459732621982bceb6ec70d267bcbc4bed1dc8d6b5bd278bf9e9fb&relay-protocol=irn&expiryTimestamp=1730810382`, since the rule will try to replace those chars with the escaped version

Here is the corrected version:

Ah, I see. The wc url uses `&`. Let's not add it then, but I suggest bringing back '<' and '>' characters, as there might be a reason they were added.

Reply:
not really needed since it isn't really utilized on the gui side if I remember correctly when I asked about this